src/loaders/tvg/tvgTvgLoader.cpp

```diff
@@ -64,8 +64,12 @@ bool TvgLoader::readHeader()
     version[TVG_HEADER_VERSION_LENGTH - 1] = '\0';
     ptr += TVG_HEADER_VERSION_LENGTH;
     this->version = atoi(version);
-    if (this->version > THORVG_VERSION_NUMBER()) {
```

Comment:
It's quite cheap, please use a cache.

Reply:
cache of what? restore THORVG_VERSION_NUMBER ?

Reply:
Yes, 'uint16_t _version'